packages/lesswrong/components/review/ReviewsLeaderboard.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import groupBy from 'lodash/groupBy';
+import sortBy from 'lodash/sortBy';
+import Card from '@material-ui/core/Card';
+import { ReviewYear } from '../../lib/reviewUtils';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    ...theme.typography.body2,
+    color: theme.palette.grey[600],
+    background: theme.palette.background.pageActiveAreaBackground,
+    padding: 16,
+    paddingBottom: 10,
+    border: theme.palette.border.commentBorder,
+    marginBottom: 16,
+    position: "sticky",
+    top: -48,
+    zIndex: theme.zIndexes.reviewLeaderboard
+  },
+  username: {
+    width: 160,
+  },
+  karma: {
+    width: 45,
+  },
+  card: {
+    width: 400,
+    maxHeight: 400
+  },
+  showAll: {
+    color: theme.palette.primary.main,
+    marginTop: 16,
+    display: "block"
+  },
+  reviews: {
+    width: 510 
+  },
+  row: {
+    marginTop: 4
+  }
+});
+
+export const ReviewsLeaderboard = ({classes, reviews, reviewYear}: {
+  classes: ClassesType,
+  reviews?: CommentsListWithParentMetadata[],
+  reviewYear?: ReviewYear
+}) => {
+  const [truncated, setTruncated] = useState<boolean>(true)
+  const { UsersNameDisplay, Row, MetaInfo, LWTooltip, CommentsNode } = Components
+
+  const userRows = Object.entries(groupBy(reviews, (review) => review.userId))
```

Comment:
If we aren't using the keys anywhere, I think this can just be `Object.values` and then you can simplify everything that follows so it doesn't need to extract the value from the tuple